### PR TITLE
Fix libgcc-ng license

### PR DESCRIPTION
**Pull request by xhochy**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by xhochy** [comment]
This won't fix the license for the most recent version. Should I send a PR to that branch, too?

**Comment by beckermr** [comment]
yes

**Comment by beckermr** [comment]
LGTM!

**Comment by xhochy** [comment]
> This won't fix the license for the most recent version. Should I send a PR to that branch, too?

Here: https://github.com/conda-forge/ctng-compilers-feedstock/pull/35

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -28,7 +28,7 @@ ctng_vendor:
 #ctng_duma:
 #  - 2.5.15
 ctng_gcc_build_number:
-  - 11
+  - 12
```
Comment (scopatz) [inline recipe/conda_build_config.yaml:31]: This probably doesn't need a build number bump

**Comment by isuruf** [comment]
@conda-forge-admin, rerender

**Comment by beckermr** [comment]
Any objections to merging?

File: recipe/conda_build_config.yaml (hunk 1 above)
Comment (beckermr) [reply recipe/conda_build_config.yaml:31]: Right. We don't ship the compilers very often so if we don't bump it now, we probably won't get a chance to actually ship the new license for a while.

**Comment by beckermr** [comment]
@conda-forge-admin restart ci

**Pull request merged**